### DISCUSSION TITLE
Will corrected scripts be updated where they are being used in Zotero?

### DISCUSSION THREAD

**Discussion by DonnaCoxBaker** [windingwind/zotero-actions-tags#239] [Q&A] [answered]
### Description

If I find and report that I'm having trouble with a script, and the script gets fixed, will updating the add-on also update the scripts? What happens if we have altered the script for some purpose? Thanks!

### Event

None

### Operation

None

### Data

```javascript
/**
 * A description of this script.
 * @author Zotero Community
 * @usage 
 * @link https://github.com/windingwind/zotero-actions-tags/discussions/
 * @see https://github.com/windingwind/zotero-actions-tags/discussions/
 */
```

**Comment by windingwind** [comment, marked as answer]
No. you'll need to manually update the script

i don't expect scripts to be updated frequently, but we can discuss how/whether to support an update.
